apps/newsletters-ui/src/app/components/DraftsTable.tsx

```diff
@@ -28,6 +31,13 @@ export const DraftsTable = ({ drafts }: Props) => {
 				accessor: 'theme',
 				sortType: 'basic',
 			},
+			{
+				Header: 'Wizard',
+				accessor: 'wizardListId',
+				Cell: ({ cell: { value } }) => (
+					<Link to={`/wizard/${value as string}`}>Edit</Link>
```

Comment:
Not sure, but you might need to update this path after re-syncing with main, because of the routing changes